### ISSUE TITLE
More "elegant" solution

### ISSUE THREAD

**Issue by ioerrorle** [StormExecute/InputSwitcherX#4]
Hey bud! Thank you for the starting point. I've played a bit with x64dbg and IDA and found that changing 1 byte in a sub-procedure does the trick too.
![изображение](https://user-images.githubusercontent.com/7738261/198750118-fd088ba4-9e75-4cf3-94fb-8bf9d24bcc75.png)
So basically shit happens in this proc marked yellow.
I moved to it and inserted return at the start, changing byte 57 to C3.
![изображение](https://user-images.githubusercontent.com/7738261/198750225-e10044e5-3011-47b6-a745-3a2f770ac9c9.png)
![изображение](https://user-images.githubusercontent.com/7738261/198750260-c3b2304f-06f0-4c28-a802-ba68e6f1eaae.png)
Here are 2 files: one (.dll.bak) is original inputswitch.dll, other is modified. You can compare them by content and see where the byte has been changed.

[inputswitch.zip](https://github.com/StormExecute/InputSwitcherX/files/9892458/inputswitch.zip)

Basically, byte sequence is:
```
48 89 5C 24 08 48 89 6C  24 10 48 89 74 24 18 57
48 83 EC 40 48 63 FA 48  8B D9 48 85 C9
```
It should define the procedure ambiguously, where 57 should be replaced with C3.
BTW, it disables this shitty Win+Space language switching key combination as well.

Thank you for the good job!
P.S. I will find the way to patch it without installing Python, shouldn't be that hard. It's just "Kill explorer.exe, find some bytes, change some bytes, start explorer.exe"

**Comment by Horozor** [comment]
@ioerrorle Hey. Can you create instruction how to change this byte for noobs. This pop-up window annoying me. Patch not working since 22h2 update, so i want to find new solution. Please help!

**Comment by dukhevych** [comment]
PLEASE HELP

**Comment by Lindsaydoodle** [comment]
НЕ ИСПОЛЬЗУЙТЕ setPatch.bat на вин 11 потому что после перезапуска приведет к черному экрану!

DO NOT USE setPatch.bat on wine 11 because after restarting it will cause a black screen!

**Comment by ioerrorle** [comment]
> @ioerrorle Hey. Can you create instruction how to change this byte for noobs. This pop-up window annoying me. Patch not working since 22h2 update, so i want to find new solution. Please help!

Hey man, sorry for the VERY late reply, you know how tough is shit nowadays, but either way you can use any HEX-editor of your choice, WinHEX is a solid start.
Just copy the inputswitch.dll somewhere, edit it, stop explorer.exe (don't forget to change the registry so it won't autostart, use google skills to finde out how to do it, hint - `HKLM/Software/Microsoft/Windows NT/CurrrentVersion/Winlogon/AutoRestartShell`), copy it without starting explorer (by powershell or cmd) and here you go.

**Comment by span4ev** [comment]
> @ioerrorle Hey. Can you create instruction how to change this byte for noobs. This pop-up window annoying me. Patch not working since 22h2 update, so i want to find new solution. Please help!
Hi. I made a tutorial for nubs, but the voiceover is in Russian, no subtitles. But maybe this will help you a little bit
https://youtu.be/EgAuZG-ii_s?si=DXEVuPF4y9gH-dsq
